PKGBUILD-git

```diff
@@ -1,35 +1,41 @@
 # Maintainer: Erik Reider <erik.reider@protonmail.com>
 pkgname=swaync-git
-_ver="0.6.1"
-pkgver="$_ver.r304.f0369e6"
+_pkgname=swaync
+pkgver="0.6.1.r304.f0369e6"
 pkgrel=1
-pkgdesc="A simple notificaion daemon with a GTK panel for checking previous notifications like other DE's"
-_pkgfoldername=SwayNotificationCenter
-url="https://github.com/ErikReider/$_pkgfoldername"
-arch=(x86_64)
-license=(GPL)
+pkgdesc="A simple notificaion daemon with a GTK panel for checking previous notifications like other DEs"
+url="https://github.com/ErikReider/SwayNotificationCenter"
+arch=(
+	'x86_64'
+	'aarch64' # ARM v8 64-bit
+	'armv7h'  # ARM v7 hardfloat
+)
+license=('GPL3')
 depends=("gtk3>=3.22" "gtk-layer-shell>=0.1" "dbus" "glib2>=2.50" "gobject-introspection>=1.68" "libgee>=0.20" "json-glib>=1.0" "libhandy>=1.4.0")
 conflicts=("swaync" "swaync-client")
 provides=("swaync" "swaync-client")
 makedepends=(vala meson git scdoc)
-source=("git+$url")
+source=("$_pkgname::git+$url")
 sha256sums=('SKIP')
 
 pkgver() {
-  cd $_pkgfoldername
-  printf "$_ver.r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+	cd $_pkgname
+	( set -o pipefail
+		git describe --long 2>/dev/null | sed 's/\([^-]*-g\)/r\1/;s/-/./g' ||
+		printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+	)
```

Comment:
Why change this? It worked perfectly fine before. This change also excludes the pkgver so the resulting version would become `swaync-git r308.a9c7af2-1` instead of `swaync-git 0.6.1.r308.a9c7af2-1`

Reply:
> I'll still need to update the file in the AUR repo to prompt an update though
That's not the intended way of performing updates on development packages though. These are intended to be automatically updated by on each -Syu if there have been now commits.

> Do you know if there's a way for GitHub to automatically do that?
GitHub has long abandoned the "git-way". Maybe you can solve that using a workflow which automatically commits releases as annotated tags as well.

Reply:
> That's not the intended way of performing updates on development packages though. These are intended to be automatically updated by on each -Syu if there have been now commits.

Makes sense :)

> Maybe you can solve that using a workflow which automatically commits releases as annotated tags as well.

Do you have any recommendations? Tagging through the terminal -> creating release -> copying the generated change-log -> adding a tag annotation could be a solution. Maybe not the best and most smooth experience though :/

Reply:
Maybe I can tinker something.

Reply:
> Do you have any recommendations?

Just remembered I have this in my .zshrc:
```shell
function tag_release {
	if [ "$1" = "" ]; then
		echo "Usage: tag_release TAG"
		exit 2;
	fi
	git shortlog $(git tag -l --sort=taggerdate | tail -1)..HEAD | git tag -a "$1" -F -
}
```

This also autogenerates a changelog and puts it into an annotated tag.

Reply:
Maybe you can also drop an upvote on https://github.com/github/feedback/discussions/4924 to let the Octocats know that it would be a good idea to have tags set as intended.

Reply:
That works perfectly! Did not know that `shortlog` existed... Works as expected :)